### PR TITLE
OPENEUROPA-1756: Fix styling of Apply button for moderation block.

### PR DESCRIPTION
**Pull request by sergepavle**
### Description

[Insert description here]

### Change log

- Added:
- Changed:
- Deprecated:
- Removed:
- Fixed:
- Security:

### Commands

```sh
[Insert commands here]

```

### CODE REVIEW

File: sass/style.scss [hunk 1]
```diff
@@ -28,3 +28,12 @@
 .form-item-text-format-value + .ecl-form-group {
   margin-top: 0;
 }
+
+// Align Apply bottom in moderation block
+.entity-moderation-form {
+  .entity-moderation-form__item {
+    button.ecl-button {
```
Comment (brummbar) [inline sass/style.scss]: Also this is overqualified, let's leave only `.ecl-button`.

Comment (sergepavle) [reply sass/style.scss]: Done

File: sass/style.scss [hunk 2]
```diff
@@ -28,3 +28,12 @@
 .form-item-text-format-value + .ecl-form-group {
   margin-top: 0;
 }
+
+// Align Apply bottom in moderation block
+.entity-moderation-form {
+  .entity-moderation-form__item {
```
[the remark this answers is not in the document]
Comment (sergepavle) [reply sass/style.scss]: Done

File: sass/style.scss [hunk 3]
```diff
@@ -28,3 +28,12 @@
 .form-item-text-format-value + .ecl-form-group {
   margin-top: 0;
 }
+
+// Align Apply bottom in moderation block
```
[the remark this answers is not in the document]
Comment (sergepavle) [reply sass/style.scss]: Done

**Pull request merged**